packages/cli-main/package.json

```diff
@@ -55,8 +55,9 @@
     "@oclif/plugin-commands": "^2.2.0",
     "@oclif/plugin-help": "^5.1.12",
     "@oclif/plugin-plugins": "^2.1.0",
+    "@shopify/cli-kit": "3.17.0",
     "@shopify/plugin-ngrok": "^3.17.0",
-    "@shopify/cli-kit": "3.17.0"
+    "abort-controller": "^3.0.0"
```

Comment:
Do we really need this?

Reply:
I can remove it but I need to refactor `renderConcurrent` first. I'll remove it in a follow up PR.